pkg/services/ngalert/models/alert_rule.go

```diff
@@ -233,11 +233,11 @@ func (alertRule *AlertRule) PreSave(timeNow func() time.Time) error {
 }
 
 func (alertRule *AlertRule) ResourceType() string {
-	return "alertRule"
+	return "alertRuleGroup"
```

Comment:
i'm not sure how easily we can make this change. any previously provisioned rules will have a provenance record of the old type string. this may cause rules which were previously provisioned to lose their provenance.

Reply:
I still have to write a migration for that.